helm-app-operator/cmd/helm-app-operator/main.go

```diff
@@ -34,6 +35,7 @@ func main() {
 
 	resource := os.Getenv(APIVersionEnvVar)
 	kind := os.Getenv(KindEnvVar)
+	// TODO(alecmerdler): Should watch all namespaces to align with OLM
 	namespace, err := k8sutil.GetWatchNamespace()
```

Comment:
It seems like this would do the right thing if you set `WATCH_NAMESPACE` to `""` (`NamespaceAll`)?

Reply:
I tried this, the Operator SDK complains and throws an error.